.changeset/config.json

```
@@ -10,6 +10,7 @@
       "@remix-run/cloudflare",
       "@remix-run/cloudflare-pages",
       "@remix-run/cloudflare-workers",
+      "@remix-run/css-bundle",
       "@remix-run/deno",
       "@remix-run/dev",
       "@remix-run/eslint-config",
```

You could consider using [glob expressions](https://github.com/changesets/changesets/blob/main/docs/fixed-packages.md#using-glob-expressions) instead to auto-include all `@remix-run/`-packages
```suggestion
       "@remix-run/*",
```